blocks/library/post-title/post-permalink.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Dashicon, ClipboardButton, Button } from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import './editor.scss';
+import { isEditedPostNew, getEditedPostAttribute } from '../../../editor/selectors';
```

Comment:
We don't want to be reaching out into the editor top-level directory. This could tie into #2473